doc/user_documentation.rst

```diff
@@ -296,13 +296,24 @@ of the names ``<coprname>:pr:<pr_id>``. ``<pr_id>`` is ID of the pull request op
 PR/commit flagging
 ^^^^^^^^^^^^^^^^^^
 
-If you would like to get your commits and pull requests in Pagure flagged with build results for each change, go to project settings in your Pagure project. Then:
+Above you configured a Pagure instance that auto-triggers Copr builds (*request*).
+How to analyze the corresponding build results from the Pagure web UI?  You can
+configure Copr to notify Pagure instance (*response*) about the build results
+using the "commit flagging" feature:
```

Comment:
Thank you for the update, makes more sense to me now.

Nit: I am still confused by the (*request*) and (*response*) brackets but the text works without them, so I would probably suggest just removing them. Possibly adding a new sentence at the end of the paragraph, saying what they used to say.

But +1 even to the current state.